proofs/hol_light/arm/proofs/mlkem_poly_reduce.ml

```diff
@@ -118,6 +118,9 @@ let overall_lemma = prove
   REWRITE_TAC[MATCH_MP lemma_rem (CONGBOUND_RULE `barred x`)] THEN
   BITBLAST_TAC);;
 
+(* NOTE: This must be kept in sync with the CBMC specification
```

Comment:
Nit: While this is the spec for the actual code, the spec for the entire function (including potential stack spilling and the final `ret`) is `MLKEM_POLY_REDUCE_SUBROUTINE_CORRECT` at the bottom. The comment would thus be slightly better placed there.

Reply:
Does this also apply to the other functions, e.g., rej_uniform?

Reply:
Yes -- I may have made the same mistake there! Can you check?

Reply:
(We're not entirely consistent here -- some functions have only one contract, but most are split in the above way, I think)

Reply:
okay - I will adapt it for all

Reply:
Great, thank you!

Reply:
Done. Please double check if I got that right.